src/Core/Prefab/GlobeView.js

```diff
@@ -24,51 +23,14 @@ import BuilderEllipsoidTile from './Globe/BuilderEllipsoidTile';
  * @property target {view} dispatched on view
  * @property type {string} initialized
  */
-/**
- * Fires when a layer is added
- * @event GlobeView#layer-added
- * @property layerId {string} the id of the layer
- * @property target {view} dispatched on view
- * @property type {string} layers-added
- */
-/**
- * Fires when a layer is removed
- * @event GlobeView#layer-removed
- * @property layerId {string} the id of the layer
- * @property target {view} dispatched on view
- * @property type {string} layers-added
- */
-/**
- * Fires when the layers oder has changed
- * @event GlobeView#layers-order-changed
- * @property new {object}
- * @property new.sequence {array}
- * @property new.sequence.0 {number} the new layer at position 0
- * @property new.sequence.1 {number} the new layer at position 1
- * @property new.sequence.2 {number} the new layer at position 2
- * @property previous {object}
- * @property previous.sequence {array}
- * @property previous.sequence.0 {number} the previous layer at position 0
- * @property previous.sequence.1 {number} the previous layer at position 1
- * @property previous.sequence.2 {number} the previous layer at position 2
- * @property target {view} dispatched on view
- * @property type {string} layers-order-changed
- */
-
 
 /**
  * Globe's EVENT
  * @property GLOBE_INITIALIZED {string} emit one time when globe is initialized
- * @property LAYER_ADDED {string} emit when layer id added in viewer
- * @property LAYER_REMOVED {string} emit when layer id removed in viewer
- * @property COLOR_LAYERS_ORDER_CHANGED {string} emit when  color layers order change
  */
 
 export const GLOBE_VIEW_EVENTS = {
     GLOBE_INITIALIZED: 'initialized',
-    LAYER_ADDED: 'layer-added',
```

Comment:
It is not so simple to withdraw it eventually without a little consultation.
A more judicious event will be `newLayerHasBeenDiplayed`